docs/src/examples/basics.md

```diff
@@ -0,0 +1,141 @@
+# Basics
+
+## Basic operations for quaternions
```

Comment:
It feels like `sign` should be somewhere discussed here as the right approach for constructing a unit quaternion.